src/histogram.ts

```diff
@@ -0,0 +1,91 @@
+/*
+	Copyright 2020 Dynatrace LLC
```

Comment:
```suggestion
	Copyright 2022 Dynatrace LLC
```
I think we put the year in which this file was first released here.

Reply:
I think this was an error by the auto linter